src/mmw/js/src/core/models.js

```diff
@@ -217,8 +217,10 @@ var TaskMessageViewModel = Backbone.Model.extend({
     message: null,
     iconClass: null,
 
-    setError: function() {
-        this.set('message', 'Error');
+    setError: function(err) {
+        var message = err === 'NO_LAND_COVER' ? 'Selected area of interest ' +
```

Comment:
Consider moving this logic outside `TaskMessageViewModel` so that it stays generic and unaware of GWLF-E business logic. Prefer something like:

```js
setError: function(message) {
  this.set('message', message || 'Error');
  this.set('iconClasses', 'fa fa-exclamation-triangle');
},
```

And supplying the detailed message in `ResultsView.onShow`